publisher/internal/analytics/ethereum/operations.go

```diff
@@ -128,9 +151,13 @@ func (rem *Removal) Process(burn WrappedEventLog) error {
 	}
 	rem.Position = *remPosition
 
-	rem.OperationBase.includeTokenPrices(&rem.Position) // 6) Getting token prices
-	rem.Position.calculatePosition()                    // 7) Save Liquidity Entry and Liquidity Pool
+	rem.OperationBase.includeTokenPrices(&rem.Position)
 
+	err = rem.calculateFeesEarned(collect.Log)
+	if err != nil {
+		return err
+	}
+	rem.Position.calculatePosition()
```

Comment:
nit: Probably the naming could be improved to `calculate` or `process`.

Reply:
Looking at `calculatePosition` and `calculate` it looks like `calculate` actually calculates the ratios, while `calculatePosition` does the rest. So perhaps it is best to rename `calculate` to `calculateRatios` as it is a subroutine used in `calculatePosition` and then use `calculate` instead of `calculatePosition`.